### ISSUE TITLE
pull_request_review_thread is a valid workflow trigger but flagged as unknown

### ISSUE THREAD

**Issue by benyitzhaki** [rhysd/actionlint#690]
## Summary

`pull_request_review_thread` is a valid GitHub Actions workflow trigger (documented in the [GitHub Actions events reference](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_review_thread)), but actionlint rejects it as an unknown webhook event:

```
unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names [events]
```

## Reproduction

```yaml
on:
  pull_request_review_thread:
    types: [resolved]
```

Running actionlint on the above reports an error even though the workflow is valid and runs correctly on GitHub.

## Root cause

`pull_request_review_thread` is absent from `all_webhooks.go`. Since that file is auto-generated by the `generate-webhook-events` scraper, the event is likely missing from the scraped GitHub docs page or structured in a way the scraper doesn't pick up.

## Expected behavior

`pull_request_review_thread` with activity types `created` and `resolved` should be accepted as a valid trigger.

## Workaround

In the meantime, the error can be suppressed with:
```
actionlint -ignore 'unknown Webhook event "pull_request_review_thread"'
```

**Comment by benyitzhaki** [comment]
Closing — I was mistaken. pull_request_review_thread is a GitHub webhook event but is not a supported GitHub Actions workflow trigger. Actionlint is correct to reject it. Sorry for the noise.
